bin/install_ossim.sh

```diff
@@ -7,27 +7,17 @@
 # Licensed under the GNU LGPL version >= 2.1.
 #
 
-if [ "$#" -lt 1 ] || [ "$#" -gt 1 ]; then
```

Comment:
@epifanio the script must stop immediately if no arguments are passed, so this should not be removed from here...